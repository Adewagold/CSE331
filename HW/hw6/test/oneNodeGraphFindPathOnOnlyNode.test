# Tests find path on one_node_graph

# Load oneNodeGraph.tsv
LoadGraph one_node_graph oneNodeGraph.tsv

# Find minimum-cost path on onlyNode
FindPath one_node_graph onlyNode onlyNode