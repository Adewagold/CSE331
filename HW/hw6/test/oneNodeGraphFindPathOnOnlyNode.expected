# Tests find path on one_node_graph

# Load oneNodeGraph.tsv
loaded graph one_node_graph

# Find minimum-cost path on onlyNode
path from onlyNode to onlyNode:
total cost: 0.000